### ISSUE TITLE
Do tourists hate my city this much?

### ISSUE THREAD

**Issue by qualiall** [Infixo/CS2-InfoLoom#2]
Not sure if bug or my play style or other mods, or if I simply don't understand the magnitude of these numbers, but it seems that my city is a no go for tourism big time.  I don't have much in the way of tourism in the city now, but still it seems like a high number.  How do I diagnose the cause of this?
![20231228150005_1](https://github.com/Infixo/CS2-InfoLoom/assets/95945450/82e4a6f4-ae16-45b2-b4c4-25b1ee3a5893)

**Comment by Infixo** [comment]
Demand factors are metrics for various "things" that CO chose to have influence on total demand. These numbers are related to the underlying "thing" but there is always some math involved, so it is not always easy to explain what each number mean.
Example.
"Students" is a metrics that is related to free college and university capacity. The game calculates all free places and then takes square root of 2*free_places. Why? Only CO knows, that is how it works.
"TouristDemand" unfortunately has some complex logic attached to it. It is related to Lodging and sums up two sub-factors.
a) usage of existing companies
b) resource production vs. consumption
So, a negative value would happen when:
a) existing Lodging companies are not utilized
b) huge Lodging production and small consumption
So, basically a) and b) measure the same thing but from different angles. Both point to the fact that you have **too many** Lodging companies and **not enough** tourists.
The game simply tells you that you should **not** build more commmercial because you have enough already.

**Comment by Infixo** [comment]
Looking at other demand factors I can say that you have heavily overinvested into Commercial zones. Negative LocalDemand means that your people don't consume so much resources and don't need so many commercial buildings (yet).
Industrial LocalInputs negative means that you are lacking in extractors.
Save your game and track down all Lodging companies and try do dezone them. See if this will reduce this negative value.
I am closing the issue because these numbers come directly from the game engine, the mod simply displays them.
